opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -144,7 +145,7 @@ import {
   isStixDomainObjectShareableContainer,
   isStixObjectAliased,
   resolveAliasesField,
-  STIX_ORGANIZATIONS_UNRESTRICTED,
+  STIX_ORGANIZATIONS_UNRESTRICTED
```

Comment:
why do we have this change ? the last ',' should be kept

Reply:
I think it's ESLint that did it, I can revert that